### PR TITLE
Trying to use a throw when invalid kv-pairs are detected

### PR DESCRIPTION
**Pull request by CounterRebel**

### CODE REVIEW

File: UT4MasterServer/Other/GameServerAttributesBsonSerializer.cs [hunk 1]
```diff
@@ -25,9 +25,7 @@ public override void Serialize(BsonSerializationContext context, BsonSerializati
 				context.Writer.WriteBoolean(key, valBool);
 			else
 			{
-				// Other kv-pairs are ignored because they are invalid
-				// TODO: Is throw more appropriate?
-				continue;
+				throw new FormatException("Invalid kv-pairs detected");
```
Comment (Saibamen) [inline UT4MasterServer/Other/GameServerAttributesBsonSerializer.cs:28]:
```suggestion
				throw new FormatException("Invalid KeyValue-pairs detected");
```

Comment (Saibamen) [inline UT4MasterServer/Other/GameServerAttributesBsonSerializer.cs:28]:
```suggestion
				throw new FormatException($"Invalid KeyValue-pairs detected. Key: {key}");
```

**Review by Saibamen — CHANGES_REQUESTED** [review]

**Comment by timiimit** [comment]
What is the reasoning for why throwing would be better?

**Comment by timiimit** [comment]
No valid reason provided for added changes.

**Pull request closed without merging**